index.html

```diff
@@ -23,62 +23,78 @@
-          <ul class="social_links">
-            <li class="social_link">
-              <a href="#"
-                ><img
-                  class="social_icon"
-                  src="./images/twitter.svg"
-                  alt="Twitter"
-              /></a>
-            </li>
-            <li class="social_link">
-              <a href="#"
-                ><img
-                  class="social_icon"
-                  src="./images/linkedin-icon.svg"
-                  alt="Linkedin"
-              /></a>
-            </li>
-            <li class="social_link">
-              <a href="#"
-                ><img class="social_icon" src="./images/m.svg" alt="M"
-              /></a>
-            </li>
-            <li class="social_link">
-              <a href="#"
-                ><img
-                  class="social_icon"
-                  src="./images/github.svg"
-                  alt="Github"
-              /></a>
-            </li>
-            <li class="social_link">
-              <a href="#"
-                ><img class="social_icon" src="./images/doit.svg" alt="Diot"
-              /></a>
-            </li>
-          </ul>
+        <div class="headline_container">
+          <h1 class="headline_title">
+            I’m Abu Raihan <br />
+            Glad to see you!
+          </h1>
+          <p class="headline_content">
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hestiate to
+            contact me.
+          </p>
+          <div class="headline_social">
+            <h3 class="social_title blue">LET’S CONNECT</h3>
+            <ul class="social_links">
+              <li class="social_link">
+                <a href="#"
+                  ><img
+                    class="social_icon"
+                    src="./images/twitter.svg"
+                    alt="Twitter"
+                /></a>
+              </li>
+              <li class="social_link">
+                <a href="#"
+                  ><img
+                    class="social_icon"
+                    src="./images/linkedin-icon.svg"
+                    alt="Linkedin"
+                /></a>
+              </li>
+              <li class="social_link">
+                <a href="#"
+                  ><img class="social_icon" src="./images/m.svg" alt="M"
+                /></a>
+              </li>
+              <li class="social_link">
+                <a href="#"
+                  ><img
+                    class="social_icon"
+                    src="./images/github.svg"
+                    alt="Github"
+                /></a>
+              </li>
+              <li class="social_link">
+                <a href="#"
+                  ><img class="social_icon" src="./images/doit.svg" alt="Diot"
+                /></a>
+              </li>
+            </ul>
+          </div>
```

Comment:
- [ ] as requested by the previous reviewer kindly try to follow the Figma design for the headline section, adding some paddings and margins will help you match the Figma design 

**your design**
![image](https://user-images.githubusercontent.com/89282221/199503970-b59183e8-a898-48be-9552-25effeace29e.png)

**Figma Design**

![image](https://user-images.githubusercontent.com/89282221/199504188-e288e99e-d96d-4344-ab18-d4f25438c7d9.png)